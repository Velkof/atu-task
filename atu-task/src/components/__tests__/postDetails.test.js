import React from 'react';
import renderer from 'react-test-renderer';
import PostDetails from '../postDetails';

const props = {
    loading: false,
    selectedPost: {
        id: 1,
        title: 'Post title',
        userId: 2,
        body: 'Post body'
    },
    selectedPostAuthor: {
        name: 'Author name'
    },
    match: {
        params: {
            id: 1
        }
    },
    getPostDetails: () => {}
}

test('<PostDetails /> renders successfully', () => {
  const tree = renderer.create(
    <PostDetails
        {...props}
    />
  ).toJSON();
  
  expect(tree).toMatchSnapshot();
});
