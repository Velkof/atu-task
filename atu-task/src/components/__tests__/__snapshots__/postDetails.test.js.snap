// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostDetails /> renders successfully 1`] = `
<div
  className="post-details"
>
  <div
    className="wrapper"
  >
    <div
      className="title"
    >
      <span>
        Post title
      </span>
    </div>
    <p
      className="description"
    >
      Post body
    </p>
    <p
      className="subtitle"
    >
      Authored by: 
      Author name
    </p>
    <div
      className="button-container"
    >
      <button
        onClick={[Function]}
      >
        Back
      </button>
    </div>
  </div>
</div>
`;
